src/DynamoCoreWpf/PublicAPI.Unshipped.txt

```diff
@@ -1465,6 +1466,9 @@ Dynamo.UI.Controls.StartPageListItem.DateModified.set -> void
 Dynamo.UI.Controls.StartPageListItem.Description.get -> string
 Dynamo.UI.Controls.StartPageListItem.Icon.get -> System.Windows.Media.ImageSource
 Dynamo.UI.Controls.StartPageListItem.IconVisibility.get -> System.Windows.Visibility
+Dynamo.UI.Controls.StartPageListItem.LoadBitmapImage(string iconPath) -> System.Windows.Media.Imaging.BitmapImage
```

Comment:
@QilongTang is it needed to add protected API's to this file?

Reply:
I dont think so

Reply:
I was fixing the errors in VisualStudio. I can remove if not needed.

Reply:
Removed the protected properties from the API.